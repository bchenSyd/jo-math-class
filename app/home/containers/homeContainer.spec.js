// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import { HomeContainer }
  from './homeContainer';

describe('HomeContainer', () => {
  it('renders correctly', () => {
    const carOfTheWeek = {
      review: 'test review',
    }
    const navigate = jest.fn();
    const tree = renderer.create(<HomeContainer carOfTheWeek={carOfTheWeek} 
          navigation={{ navigate }} isFetching={false} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
