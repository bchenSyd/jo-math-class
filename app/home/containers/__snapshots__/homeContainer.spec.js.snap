// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContainer renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "flex-start",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "flex": 1,
        }
      }
    >
      Car of the week
    </Text>
    <View
      style={
        Object {
          "flex": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
      >
        test review
      </Text>
    </View>
  </View>
  <View>
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          goto Playground
        </Text>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#0091EA",
          "marginTop": 10,
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 17,
            "fontWeight": "500",
            "padding": 24,
          }
        }
      >
        goto Animation
      </Text>
    </View>
  </View>
</View>
`;
